cmd/validator/validator.go

```diff
@@ -0,0 +1,5 @@
+package validator
+
+import "github.com/spf13/cobra"
```

Comment:
added this on the same level as the info and runner packages rather than cmd to avoid circular dependencies.